src/applications/personalization/rated-disabilities/components/RatedDisabilityHeader.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class RatedDisabilityHeader extends React.Component {
+  render() {
+    return (
+      <>
```

Comment:
Do we need to use a Fragment here since there's only 1 child (e.g. `<header>`)?

Reply:
We probably don't even need this component, since the header is actually gonna be static :stuck_out_tongue_winking_eye: I'll just get rid of it.